crates/syn-solidity/src/attribute/function.rs

```diff
@@ -207,11 +197,15 @@ impl Parse for FunctionAttribute {
 }
 
 impl From<VariableAttribute> for FunctionAttribute {
+    /// Converts a variable attribute to its corresponding function attribute.
+    ///
+    /// - `constant` -> `pure`
+    /// - `immutable` -> `view`
```

Comment:
both `constant` and `immutable` correspond to code regions and `codecopy` commands, no? shoudn't they be the same?

Reply:
Doesn't really matter, just needed a way to differentiate the two